lib/rubocop/ast/builder.rb

```diff
@@ -20,6 +20,13 @@ class Builder < Parser::Builders::Default
       NODE_MAP = {
         and:          AndNode,
         alias:        AliasNode,
+        arg:          ArgNode,
+        blockarg:     ArgNode,
+        kwarg:        ArgNode,
+        kwoptarg:     ArgNode,
+        kwrestarg:    ArgNode,
+        optarg:       ArgNode,
+        restarg:      ArgNode,
```

Comment:
there's also a `shadowarg`

Reply:
Can you give me an example of it?

Reply:
Never mind [found it](https://github.com/whitequark/parser/blob/master/doc/AST_FORMAT.md#block-shadow-arguments). Same question then as `forward_arg`, should it be added to the `Node::ARGUMENT_TYPES` set?

Reply:
I always forget about these. I've never used them, or seen them in the wild!

> Same question then as `forward_arg`, should it be added to the `Node::ARGUMENT_TYPES` set?

Yes, it should.